.gitignore

```diff
@@ -1,3 +1,4 @@
 rosetta-ethereum
 ethereum-data
 cli-data
+.idea
```

Comment:
This should be project specific gitignore, not dev env specific. Please move this to your global gitignore.